ChatSecure/Classes/View Controllers/OTRMessagesViewController.m

```diff
@@ -2397,15 +2408,71 @@ - (nullable NSArray *)supplementaryViewsForCellAtIndexPath:(NSIndexPath *)indexP
     return nil;
 }
 
-- (void)populateUnknownSenderCell:(OTRMessagesUnknownSenderCell *)cell withMessage:(id<OTRMessageProtocol,JSQMessageData>)message {
+- (void)populateUnknownSenderCell:(OTRMessagesUnknownSenderCell *)cell withMessage:(id<OTRMessageProtocol,JSQMessageData>)message indexPath:(NSIndexPath *)indexPath {
```

Comment:
Please move this to an instance method on `OTRMessagesUnknownSenderCell`. Although it's kind of verbose and not exactly ideal, you should be able to pass in all of the dependencies individually.

We really should try to prevent adding any more lines to OTRMessagesViewController, and aim for net-negative over time.